package.json

```diff
@@ -41,18 +41,19 @@
         "eslint-config-fluid": "1.2.0-dev.20180213T130610Z.40e6281",
         "fluid-grunt-eslint": "18.1.2",
         "gpii-testem": "2.1.1-dev.20180320T102716Z.5caff1c",
-        "grunt": "1.0.1",
+        "grunt": "1.0.2",
         "grunt-contrib-clean": "1.1.0",
         "grunt-contrib-compress": "1.4.3",
         "grunt-contrib-concat": "1.0.1",
         "grunt-contrib-copy": "1.0.0",
         "grunt-contrib-stylus": "1.2.0",
-        "grunt-contrib-uglify": "3.1.0",
+        "grunt-contrib-uglify": "3.3.0",
         "grunt-contrib-watch": "1.0.0",
         "grunt-jsonlint": "1.1.0",
         "grunt-modulefiles": "0.3.1",
         "grunt-shell": "2.1.0",
-        "lodash": "4.17.4",
+        "istanbul": "git://github.com/the-t-in-rtf/istanbul#828eb930add7bd663ca20649a917f3f3829e48c5",
```

Comment:
Is this being used by Infusion?

Reply:
I believe this is no longer necessary.  The improvements I submitted in that pull (and resubmitted when they relocated the project) have been merged and released.  Everything we use (nyc, testem) should be able to use a released version rather than our work.